Packages.props

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I would like to try and keep the root as clean as possible so prefer this to go into the eng directory.  According to the docs at https://github.com/Microsoft/MSBuildSdks/tree/master/src/CentralPackageVersions you can set the `CentralPackagesFile` in the props file to put the packages file somewhere else.